server/models.py

```diff
@@ -53,8 +111,22 @@ class Membership(models.Model):
 class Vaccination(models.Model):
     player = models.OneToOneField(Player, on_delete=models.CASCADE)
     is_vaccinated = models.BooleanField()
-    vaccination_name = models.CharField(max_length=100, blank=True)
-    vaccination_certificate = models.FileField(upload_to="vaccination_certificates/", blank=True)
+
+    class VaccinationName(models.TextChoices):
+        COVISHIELD = "CVSHLD", _("Covishield")
+        COVAXIN = "CVXN", _("Covaxin")
+        MODERNA = "MDRN", _("Moderna")
+        SPUTNIK = "SPTNK", _("Sputnik")
+        JOHNSON_AND_JOHNSON = "JNJ", _("Johnson & Johnson")
```

Comment:
Why are we using these abbreviations? Is it because the char length is too small? Would it be better to just use the full names? Or it doesn't matter? :thinking: